server/controllers/follow.controllers.js

```diff
@@ -104,14 +112,14 @@ const followController = {
           followee_id: unFolloweeId,
         },
       });
+      const userDetails = profileHelper.profiler(findFollowee);
       return res.status(200).json({
-        status: 200,
-        message: 'Successfully unfollowed user',
+        message: 'You have successfully unfollowed user',
```

Comment:
`You have successfully unfollowed this user`

Reply:
Noted and fixed